app/views/govuk_component/organisation_logo.raw.html.erb

```diff
@@ -11,7 +11,15 @@
     <div class="<%= logo_container_class %>">
   <% end %>
 
-  <span><%= raw organisation[:name] %></span>
+  <% if organisation[:image] %>
+    <img
+      class="logo-image"
+      src="<%= organisation[:image][:url] %>"
+      <% if organisation[:image][:alt_text] %>alt="<%= organisation[:image][:alt_text] %>"<% end %>
```

Comment:
The name that comes through is the formatted one. I'd like to avoid stripping out the HTML in favour of specifying. It should always be provided – could make it required?